rds.tf

```diff
@@ -81,7 +81,7 @@ resource "aws_db_instance" "keycloak-database-engine" {
   allocated_storage      = 20
   max_allocated_storage  = 100
   engine                 = "mariadb"
-  engine_version         = "10.5.12"
+  engine_version         = "10"
```

Comment:
Per [here](https://docs.aws.amazon.com/AmazonRDS/latest/UserGuide/MariaDB.Concepts.VersionMgmt.html):

> For MariaDB, version numbers are organized as version X.Y.Z. In Amazon RDS terminology, X.Y denotes the major version, and Z is the minor version number. For Amazon RDS implementations, a version change is considered major if the major version number changes, for example going from version 10.5 to 10.6. A version change is considered minor if only the minor version number changes, for example going from version 10.6.5 to 10.6.7. 

Given that info I think it's better to set this as `10.5`.